BioMed.Api/BioMed.Domain/Entities/Treatment.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace BioMed.Domain.Entities
+{
+    public class Treatment : EntityBase
+    {
+        public string? Prescription { get; set; }
+
```

Comment:
Why is there a spacebar here

Reply:
To separate property with navigation property.